telepot/loop.py

```diff
@@ -44,8 +44,8 @@ def __init__(self, bot, on_update):
         self._bot = bot
         self._update_handler = on_update
 
-    def run_forever(self, relax=0.1, timeout=20, allowed_updates=None):
-        offset = None  # running offset
+    def run_forever(self, relax=0.1, timeout=20, allowed_updates=None, offset=None):
+        offset = offset  # running offset
```

Comment:
The statement `offset = offset` is useless.

Reply:
That's OK. I can add it myself. I will also mirror it in the async version. Expect that in the next release. Thanks to @HoskeOwl

Reply:
Sorry. I mean I can add the docstring and stuff.

And @das7pad is correct, `offset = offset` is useless. But that's OK too. I can edit that. The suggestion of letting user pass in an `offset` argument sounds good.